Doc/whatsnew/2.7.rst

```diff
@@ -2540,6 +2540,21 @@ exemption allowing new ``-3`` warnings to be added in any Python 2.7
 maintenance release.
 
 
+Two new environment variables for debug mode
+--------------------------------------------
+
+In debug mode, the ``[xxx refs]`` statistic is not written by default, the
+:envvar:`PYTHONSHOWREFCOUNT` environment variable now must also be set.
+(Contributed by Victor Stinner; :issue:`31733`.)
+
+When Python is compiled with ``COUNT_ALLOC`` defined, allocations counts are no
+more dumped by default anymore: the :envvar:`PYTHONSHOWALLOCCOUNT` environment
```

Comment:
Nit: "no more dumped" isn't grammatically correct. 

Perhaps change:

`allocations counts are no more dumped by default anymore`

to:

`allocation counts are no longer dumped by default`

Cheers

Reply:
I'm sorry. It's not the first time that someone reports me this grammar mistake, and I continue to make it... Thank you for your review. Since english is not my first language, I appreciate such feedback ;-)

Reply:
No need to apologize. Thanks for all you do!